src/config/config.js

```diff
@@ -1,17 +1,31 @@
+// import Message from "./message.js";
+// import Event from "./event.js";
+import { eventsArray } from "./event.js";
+
 export default class Config {
-  constructor(frequency = 5000, endHour = 23, endMinute = 55) {
-      this.frequency = frequency;
-      this.endHour = endHour;
-      this.endMinute = endMinute;
+  static BuddyProfilePicSrc = "./assets/images/buddy-profile-pic-cat.png";
+
+  constructor(frequency = 5000, endHour = 23, endMinute = 59, userName = "buddy") {
+    this.frequency = frequency;
+    this.endHour = endHour;
+    this.endMinute = endMinute;
+    this.eventsArray = eventsArray;
+    this.userName = userName;
+  }
+
+  setFrequency(userInput) {
+    this.frequency = userInput;
   }
 
-  updateFrequency(userInput) {
-      this.frequency = userInput;
+  getFrequency() {
+    return this.frequency;
   }
```

Comment:
Do we even need these methods?
I don't think you need. Just put the var public then can already.
*I mean, it is already public. Anyone who uses this class do not even need to use these 2 methods to assign / get the value of `this.frequency`